src/ui/common/src/components/MultiFileViewer/index.tsx

```diff
@@ -44,17 +44,14 @@ const MultiFileViewer: React.FC<Props> = ({
     );
   };
 
-  let hasFiles = files && Object.keys(files).length > 0;
-
+  const hasFiles = files && Object.keys(files).length > 0;
   let selected = files;
   if (hasFiles) {
     const pathList = selectedFile.split('/').splice(1);
 
     pathList.forEach((section) => {
       if (Object.keys(selected).includes(section)) {
         selected = selected[section];
-      } else {
-        hasFiles = false;
```

Comment:
@agiron123 I'm not sure why you added this originally but this has caused the viewer not showing any content when there's no file selected

Reply:
I was trying to get this to work with showing the SQL query that was used in an operator, but that ended up taking a bit too long, so i chose another approach. 

I think I just forgot to take this back out.